test/functionalTest/cases/0967_v2_query_op/illegal_chars_in_url_param.test

```diff
@@ -79,20 +77,6 @@ echo
 echo
 
 
```

Comment:
Ah, yes, of course. I had that in mind 15 minutes beofre implementing the change. Then I forgot!